tox.ini

```diff
@@ -63,6 +64,24 @@ commands =
     pytest --import-mode=append {posargs}
 
 
+[testenv:torch-tests]
+# this environement runs the tests of the equistore-torch Python package
+passenv = *
+deps =
+    pytest
+    numpy
+    torch
```

Comment:
How does this tox env get the torch cpu wheels? I don't see anywhere here an index url for the cpu specified.